### DISCUSSION TITLE
Supply Drops are empty

### DISCUSSION THREAD

**Discussion by marceloft** [TEHE-Studios/ExpandedHelicopterEvents#295] [Q&A]
The supply drops are coming empty and the drop itself is listed as a vehicle. Any idea what could be causing this? Hosted server.

**Comment by sharpgolden** [comment]
Not a dev, but drops are vehicles, just without any wheels and doors. Cant help you with empty drops though. You can try to find zombies with EHE gear and items in them, if they are present, then its a really specific bug that only devs can help you with. Gl

**Comment by Chuckleberry-Finn** [comment]
Are you using any mods that makes loot more rare? Those mods usually just mess with loot tables.

**Reply by marceloft** [reply]
hmm... that's it then... is there any way to organize the heli loot table manually?

**Reply by Chuckleberry-Finn** [reply]
You should be able to simply move EHE to load after/before - not sure which would help here.

But, let me know what loot mods you're using, and I can try to patch the issue.

**Reply by marceloft** [reply]
I tried changing priorities but it didn't work...
The only mod about loot i was using is More Loot Settings ( https://steamcommunity.com/sharedfiles/filedetails/?id=2812491628 )
